src/mono/browser/build/WasmApp.InTree.props

```diff
@@ -12,6 +12,8 @@
     <EnableTargetingPackDownload>false</EnableTargetingPackDownload>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
     <EMSDK_PATH Condition="'$(EMSDK_PATH)' == '' and '$(MonoProjectRoot)' != ''">$([MSBuild]::NormalizeDirectory($(MonoProjectRoot), 'browser', 'emsdk'))</EMSDK_PATH>
+    <EmscriptenNodeToolsPath>$([MSBuild]::NormalizeDirectory( $([System.IO.Path]::Combine($(NuGetPackageRoot), 'runtime.$(_portableHostOS)-$(BuildArchitecture).microsoft.netcore.runtime.wasm.node.transport', '$(NodePackageVersion)', 'tools', '$(_portableHostOS)-$(BuildArchitecture)')) ))</EmscriptenNodeToolsPath>
```

Comment:
How is this not being executed?
linux:
`Specified Emscripten sdk at $(EMSDK_PATH)=/__w/1/s/src/mono/browser/emsdk/ is missing some paths: $(EmscriptenNodeToolsPath)=/__w/1/s/src/mono/browser/emsdk/node/bin/`
windows:
`Specified Emscripten sdk at $(EMSDK_PATH)=D:\a\_work\1\s\src\mono\browser\emsdk\ is missing some paths: $(EmscriptenPythonToolsPath)=D:\a\_work\1\s\src\mono\browser\emsdk\python\`